core/src/main/java/io/th0rgal/oraxen/api/events/custom_block/OraxenBlockDropLootEvent.java

```diff
@@ -0,0 +1,55 @@
+package io.th0rgal.oraxen.api.events.custom_block;
+
+import io.th0rgal.oraxen.mechanics.provided.gameplay.custom_block.CustomBlockMechanic;
+import io.th0rgal.oraxen.utils.drops.DroppedLoot;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class OraxenBlockDropLootEvent extends Event {
```

Comment:
Rename to OraxenCustomBlockDropLootEvent
Also add sub-events for noteblock & stringblock mechanics

Reply:
Done